eclair-core/src/main/scala/fr/acinq/eclair/api/EclairApi.scala

```diff
@@ -0,0 +1,190 @@
+package fr.acinq.eclair.api
+
+import akka.util.Timeout
+import akka.pattern._
+import fr.acinq.bitcoin.Crypto.PublicKey
+import fr.acinq.bitcoin.{ByteVector32, MilliSatoshi, Satoshi}
+import fr.acinq.eclair.{Kit, ShortChannelId}
+import fr.acinq.eclair.io.{NodeURI, Peer}
+import akka.actor.{Actor, ActorRef, ActorSystem, Props}
+import fr.acinq.eclair.channel._
+import fr.acinq.eclair.db.{NetworkFee, Stats}
+import fr.acinq.eclair.io.Peer.{GetPeerInfo, PeerInfo}
+import fr.acinq.eclair.payment.PaymentLifecycle._
+import fr.acinq.eclair.payment.{PaymentLifecycle, PaymentReceived, PaymentRequest}
+import fr.acinq.eclair.router.{ChannelDesc, RouteNotFound, RouteRequest, RouteResponse}
+import fr.acinq.eclair.wire.{ChannelAnnouncement, ChannelUpdate, NodeAddress, NodeAnnouncement}
+import scodec.bits.ByteVector
+import scala.concurrent.duration._
+import scala.concurrent.Future
+
+trait EclairApi {
```

Comment:
`Eclair` is too generic.

> @dpad85 could this api be used by Eclair Mobile?

Not with the way it's done here, because it's a field of `NewService` which is removed on android. It could be used if it was a parameter of `fr.acinq.eclair.Kit`, which is returned by `Setup#bootstrap()`.

Since `EclairAPI` is in fact a helper interface over the application `Kit`, why not move all these utility functions to `Kit`. It would be the standard way of interacting with the node, through the Kit.

We could have:

`trait EclairInterface` listing the helper functions

`Kit` implementing `EclairInterface` 

`NewService` would inherit all necessary methods. And I could use this helper functions from Eclair Mobile.